live/global.ini

```diff
@@ -65136,59 +65136,59 @@ searchbody_danger_0001=Wir wären nachlässig, wenn wir dich nicht daran erinner
-searchbody_desc_0005=Auf diesem Gebiet wird kein fröhliches Ergebnis erwartet. ~mission(TargetName) wurde zuletzt gehört, als sie an Bord einer "~mission(Ship)" gingen. Das gleiche Schiff wurde als zerstört gemeldet. Jetzt möchte die Familie bestätigen, was passiert ist, eine Möglichkeit oder die andere. Da kommst du ins Spiel. Geh, suche nach ~mission(Location), finde ~mission(TargetName|Last) und hoffentlich können wir ihnen ein wenig Ruhe verschaffen. ~mission(Timed)~mission(Danger)
+searchbody_desc_0001=Die Familie von ~mission(TargetName) hofft, einen Schlussstrich ziehen zu können. Es gibt zwar eine Bestätigung, dass die ~mission(Ship), auf der ~mission(TargetName|First) gedient hat, die Reise nicht überlebt hat, aber die Familie möchte sicher wissen, was passiert ist. ~mission(Contractor) sucht jemanden mit Taktgefühl und Diskretion, ~mission(Location) zu untersuchen und zu sehen, ob sie das Schicksal von ~mission(TargetName|First) erfahren können. ~mission(Timed)~mission(Danger)Deine Hilfe in dieser Sache ist willkommen.
+searchbody_desc_0002=Vor kurzem hatte eine ~mission(Ship) Pech und wurde zerstört. Seitdem hat niemand mehr ~mission(TargetName) gehört oder gesehen, und wir gehen davon aus, dass er an Bord war, als die ~mission(Ship) ihr Schicksal ereilte. Wir suchen einen Auftragnehmer, der ~mission(Location) untersucht und feststellt, ob ~mission(TargetName|Last) dort ist. ~mission(Timed)~mission(Danger)
+searchbody_desc_0003=Wir brauchen einen Auftragnehmer, um eine Person, ~mission(TargetName), zu finden, die zuletzt bei der Arbeit an Bord einer ~mission(Ship) gesehen wurde, das vor kurzem als zerstört gemeldet wurde. ~mission(Timed)~mission(Danger)Die Bezahlung erfolgt, sobald wir eine Bestätigung über den Aufenthaltsort von ~mission(TargetName|Last) erhalten.
+searchbody_desc_0004=~mission(Contractor) sucht derzeit einen Auftragnehmer, der uns hilft, den Status von ~mission(TargetName) zu bestätigen. Nach ersten Recherchen gehen wir davon aus, dass ~mission(TargetName|First) an Bord einer ~mission(Ship) diente, die einen tragischen Unfall hatte. Wir möchten, dass du ~mission(Location) untersuchst, damit wir feststellen können, ob ~mission(TargetName|First) tatsächlich an Bord umgekommen ist, und damit wir die Eltern informieren können. ~mission(Timed)~mission(Danger)
+searchbody_desc_0005=Wir erwarten kein gutes Ergebnis in diesem Fall. ~mission(TargetName) hat zuletzt von sich hören lassen, als sie eine ~mission(Ship) geentert hat. Dasselbe Schiff wurde als zerstört gemeldet. Jetzt will die Familie wissen, was genau passiert ist. Hier kommst du ins Spiel. Suche ~mission(Location), finde ~mission(TargetName|Last) und wir können ihnen hoffentlich ein wenig Seelenfrieden geben. ~mission(Timed)~mission(Danger)
```

Comment:
Ich glaube das selbe schreibt man getrennt

Reply:
Zusammen in dem Fall